ansible/group_vars/mofed_2410_1140.yml

```diff
@@ -1,2 +1,4 @@
 mellanox_ofed_version: "24.10-1.1.4.0"
 use_hpc_artifact: false
+mft_version: "mft-4.26.0-93"
```

Comment:
This metadata is missing for a few other `mofed_*` groups, so it should probably be updated in those places too. It's also possible I'm just holding this wrong and need to include these vars in some other way - I just noticed they were populated for some groups but not for others.